addons/website_sale/controllers/main.py

```diff
@@ -866,6 +862,7 @@ def cart_update_json(
             **kw
         )
 
+        values['notification_info'] = self._get_cart_notification_information(order, [values['line_id']])
```

Comment:
I can't, this is returned by `sale.order._cart_update`, which does not support multiples lines at the same time (although it could but it not the purpose of this task)